lesson_10/lesson_10_mihaela_vaida/obj_properties_mihaela.py

```diff
@@ -0,0 +1,42 @@
+class Student(object):
+
+    def __init__(self, first_name, last_name, age):
+        self.first_name = first_name
+        self.last_name = last_name
+        if type(age) is not int:
```

Comment:
Let's go one step further and try not having duplicated code

Reply:
I think  I made it 
Property age of Student @age.setter def age(self, age: {setter, __lt__}) -> Any